pkgs/by-name/sl/sleek-grub-theme/package.nix

```diff
@@ -0,0 +1,30 @@
+{pkgs, lib, ...}:
+pkgs.stdenv.mkDerivation rec {
+
+  pname = "sleek-grub-theme";
+  version = "unstable-2022-06-04";
+
+  src = pkgs.fetchFromGitHub ({
+    owner = "sandesh236";
+    repo = "sleek--themes";
+    rev = "981326a8e35985dc23f1b066fdbe66ff09df2371";
+    sha256 = "sha256-yD4JuoFGTXE/aI76EtP4rEWCc5UdFGi7Ojys6Yp8Z58=";
+  });
+
+  banner = "Grub Bootloader";
+
+  installPhase = ''
+    mkdir -p $out/
+
+    cp -r 'Sleek theme-dark'/sleek/* $out/
+    sed -i "s/Grub Bootloader/${banner}/" $out/theme.txt
+  '';
+
+  meta = {
+    description = "Configurable, modern theme for the GRUB bootloader, containing backgrounds, icons, fonts and styling";
+    homepage = "https://github.com/sandesh236/sleek--themes";
+    license = lib.licenses.gpl3;
```

Comment:
`gpl3Plus` or `gpl3Only`?

Reply:
Sorry, I double checked it , it's mit, and I fixed it.